docker/releases/build_stable.sh

```diff
@@ -1,13 +1,60 @@
-#!/bin/bash
-
-set -e
+#!/usr/bin/env bash
 
 # Need to log in to Docker desktop before docker push will succeed.
 # e.g. `docker login`
-# Log in account should be 'algorand'.
+# Login name is "algorand".
+
+GREEN_FG=$(tput setaf 2)
+RED_FG=$(tput setaf 1)
+END_FG_COLOR=$(tput sgr0)
+
+NAME=stable
+NETWORK=""
+
+while [ "$1" != "" ]; do
+    case "$1" in
+        -n|--name)
+            shift
+            NAME="$1"
+            NETWORK="-g $1"
+            ;;
+        *)
+            echo "$RED_FG[$0]$END_FG_COLOR Unknown option $1."
+            exit 1
+            ;;
+    esac
+    shift
+done
```

Comment:
Can probably skip the parameters, the only valid parameter is "testnet" anyway. I think you can just tell this to build mainnet/testnet.

Reply:
Ok, will change.

Reply:
You still need to check that this script received either mainnet or testnet parameter.. otherwise, just exit the script with some instructions about the correct usage.

Reply:
Will check for only those two strings.

Reply:
Will check for only those two strings.